osfclient/cli.py

```diff
@@ -65,6 +69,39 @@ def _setup_osf(args):
 
     return OSF(username=username, password=password)
 
+def init(args):
+    """Initialize or edit an existing .osfcli.config file
+
+    """
+    # reading existing config file, convert to configparser object
+    config = config_from_file()
+    config_ = configparser.ConfigParser()
```

Comment:
Makes sense. Also you probably use `osf init` when you are new -> don't yet have environment variables.